src/tests.proj

```diff
@@ -8,6 +8,7 @@
     <CoverageReportDir>$(ArtifactsDir)coverage</CoverageReportDir>
     <!-- For UAP we are using an APPX that is registered with a unique ID. Because of that we need to run tests sequentially -->
     <SerializeProjects Condition="'$(BuildingUAPVertical)' == 'true' or '$(Coverage)' == 'true'">true</SerializeProjects>
+    <TraversalBuildDependsOn Condition="'$(BuildingUAPVertical)' == 'true' and '$(BuildAllConfigurations)' != 'true'">MakeCommonResourcesPriFile;$(TraversalBuildDependsOn)</TraversalBuildDependsOn>
```

Comment:
Did you consider putting this in https://github.com/dotnet/corefx/blob/master/src/pretest.builds?

Reply:
That's probably a better place for it but I will keep it as is to unblock CI. We can move it later.

Reply:
Was thinking the same